src/parser/ast.ml

```diff
@@ -1,18 +1,19 @@
 (** Abstract syntax tree representing ocamldoc comments *)
 
 (** This is a syntactic representation of ocamldoc comments. See
-    {{:https://ocaml.org/releases/4.12/htmlman/ocamldoc.html}The manual} for a detailed
-    description of the syntax understood. Note that there is no attempt at semantic
-    analysis, and hence these types are capable of representing values that will
-    be rejected by further stages, for example, invalid references or headings that
-    are out of range. *)
+    {{:https://ocaml.org/releases/4.12/htmlman/ocamldoc.html} The manual} for a
```

Comment:
The space in `{{!ref} content}` and `{{:link} content}` seems a bit weird to me!

Reply:
Also fixed: https://github.com/ocaml-ppx/ocamlformat/pull/2608